python/ray/services.py

```diff
@@ -945,17 +945,30 @@ def _start_redis_instance(executable,
             if " " in password:
                 raise ValueError("Spaces not permitted in redis password.")
             command += ["--requirepass", password]
-        command += (
-            ["--port", str(port), "--loglevel", "warning"] + load_module_args)
+        command += ["--port", str(port), "--loglevel", "warning"]
+        command += ["--supervised", "upstart"]
+        command += load_module_args
+        wait_for_sigstop = sys.platform != "win32"
+        env_updates = {}
+        if wait_for_sigstop:
+            env_updates.update(UPSTART_JOB="")
```

Comment:
can we link to some documentation about this in redis repo?

Reply:
Done